modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -85,6 +104,23 @@ by the `setup_libtorch.sh` script.
 
 !alert-end!
 
+For conda-based installations the user can link to the conda-based libtorch libraries
+using the approach above (using a typical installation path within conda):
+
+
+./configure --with-libtorch=/yourcondadirector/mambaforge3/envs/moose-torch/lib/python3.10/site-packages/torch
```

Comment:
Should we manually update the python version in this command whenever moose python version changes?

Reply:
Unfortunately, that is something we'll have to do. I can't add the automatic [!packages!minimum_python] within a code block.

Reply:
That's a bummer. Maybe create an issue for now. Intuitively, this has to work as it's just preprocessing the markdown files.

Reply:
https://github.com/idaholab/moose/issues/27702